src/Core/AmbientWeatherNetwork/ApiClient.cs

```diff
@@ -0,0 +1,32 @@
+﻿using Common.Observe;
+using Common;
+using Serilog;
+using Flurl.Http;
+using Core.AmbientWeatherNetwork.Dto;
+
+namespace Core.AmbientWeatherNetwork;
+
+public interface IApiClient
+{
+	Task<ICollection<UserDevice>> GetLatestFromDevicesAsync(string apiKey, string applicationKey);
+}
+
+public class ApiClient : IApiClient
+{
+	private const string Host = "https://rt.ambientweather.net";
```

Comment:
Perhaps provide a way to override this via config as it seems the subdomain has changed a couple times in the past. And some users have reported certain subdomains as being flakier than others.